src/test/java/org/folio/rest/impl/InventoryStorageMock.java

```diff
@@ -0,0 +1,78 @@
+package org.folio.rest.impl;
+
+import static org.junit.Assert.fail;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServer;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.BodyHandler;
+import org.apache.log4j.Logger;
+
+public class InventoryStorageMock {
```

Comment:
Can we make it generic Okapi mock so that it can be reused/extended if we need to use other modules?

Reply:
Done. Thanks.